src/inspector/models/notation/notes/beams/beamsettingsmodel.h

```diff
@@ -69,13 +74,19 @@ class BeamSettingsModel : public AbstractInspectorModel
     PropertyItem* beamHeightRight() const;
     bool isBeamHeightLocked() const;
 
+    PropertyItem* stemDirection() const;
+    PropertyItem* crossStaffMove() const;
+    bool iscrossStaffMoveAvailable() const;
```

Comment:
Nitpick:
```suggestion
    bool isCrossStaffMoveAvailable() const;
```
(sorry 😄)